src/main/java/net/sf/log4jdbc/StatementSpy.java

```diff
@@ -50,7 +50,7 @@ public class StatementSpy implements Statement, Spy
         case "true": return true;
         case "false": return false;
         default:
-          String msg = String.format("IGNORING INVALID VALUE %s FOR %s. USING FALSE.", v, SQL_SHOW_PARAMS_KEY);
+          var msg = String.format("IGNORING INVALID VALUE %s FOR %s. USING FALSE.", v, SQL_SHOW_PARAMS_KEY);

```

Comment:
is this the reason for the failure? I don't know enough about `var` to know what this is about.

Reply:
Not sure if you are asking why I used `var`? Nothing specific except that I wanted to make sure that the JDK 11 features are compiling ok. Besides, it's succinct.

Reply:
the build error implied that `var` was being interpreted as a bare symbol rather than a keyword

Reply:
Thats because of the source and target version which was reset to 1.8. It works now.